terraform/environment/dns.tf

```diff
@@ -0,0 +1,29 @@
+variable "subdomains" {
+  type = list(string)
+  default = [
+    "nginz-https",
+    "nginz-ssl",
+    "webapp",
+    "assets",
+    "account",
+    "teams"
+  ]
+}
+
+variable "create_spf_record" {
+  type = bool
+  default = true
+}
+
+module "aws-dns-records" {
+  count =  min(1, var.kubernetes_node_count)
+
+  source = "../modules/aws-dns-records"
+
+  environment = var.environment
+  subdomains = var.subdomains
+  # root_domain = var.root_domain
+  zone_fqdn = var.root_domain
+  ips = [ for node in local.kubernetes_nodes: node.ipaddress ]
```

Comment:
issue: I couldnt find this var anywhere? Am I holding it wrong?

thought: this seems to tighten the relationship between the hetzer-kubernetes module and this one again. Are we able to find a more generic way to obtain and feed ips into the module?

Reply:
The local var is created in `inventory.tf`, as I understand it the more generic way is to use outputs but it seems we can't because of a limitation in terraform when using 2 different directory trees.